app.py

```diff
@@ -73,8 +73,8 @@ def __init__(self, pExchange, pTicker):
 
 
 PAIRS = []  # Array containing all pairs
-E_GDAX = Exchange("GDAX", ["ETH-USD", "ETH-BTC", "BTC-USD",
-                           "LTC-USD", "LTC-BTC", "ETH-EUR", "BTC-EUR", "LTC-EUR", "BCH-USD", "BCH-BTC", "BCH-EUR"], 0)
+E_GDAX = Exchange("GDAX", ["ETH-USD", "ETH-EUR", "ETH-BTC",
```

Comment:
We had the previous order to give most desired charts at top.
I mean a person (e.g. me) who wants to see EUR Chart prefers havin EUR all together.
While from a trading perspective with Price tied to BTC ETH-USD and BTC-USD should be right after each otehr

Reply:
I see. I thought it was just random and would be more organized if they are grouped by the same pair first, so I changed it. With the checkboxes to show/hide I think it's better like this anyway, instead of searching the list for the pair you want.

Reply:
@pmaji what´s your opinion on this?

Reply:
I selfishly like the idea having the ETH pairs first, but tbh that was when I first built this thing with the only intention being that I would use this personally. Now that this has become a community project, it makes sense to humbly step back and simply order by most traded. 

With that assumption out of the way, the 1st degree order (based on popularity) should be as follows:

-BTC
-ETH
-LTC
-BCH

Next we have to decide within-currency pair ordering. That needs to be done based on the customers we serve, who are predominantly trading in USD-denomination. As such, the order should be

-USD pairing
-coin/coin pairing if dominant
-EUR pairing

Final order below:

-BTC-USD
-BTC-EUR
-BTC-GBP
-ETH-USD
-ETH-BTC
-ETH-EUR
-LTC-USD
-LTC-BTC
-LTC-EUR
-BCH-USD
-BCH-BTC
-BCH-EUR

Reply:
I still think ETH pairs should be first (and I also think exchanges should make *-ETH pairs instead of *-BTC). ETH is the future and I think small projects like this starting to prioritize it over BTC is the right step forward. Think about it some more please :-)

Reply:
Haha one step ahead of you @mifunetoshiro :) Thought the same thing right after typing it.